main.cpp

```diff
@@ -483,8 +478,8 @@ std::vector<int> CVSequencer::arp(int numStepsPressed)
     std::vector<int> arp; 
     for (int i = 0; i <= numStepsPressed; ++i)
     {
-        int stepNum;
-        std::cin >> stepNum;
+        int stepNum = 1;
+        std::cout << stepNum;
```

Comment:
you're going to want to add a newline after this `stepNum` when you call this function.  Otherwise all of your `stepNum`'s will print on the same line, without spaces.